functions/serverless-functions-getting-started.adoc

```diff
@@ -40,7 +40,7 @@ include::modules/serverless-functions-creating-deploying-invoking.adoc[leveloffs
 * link:https://docs.openshift.com/container-platform/latest/registry/securing-exposing-registry.html#securing-exposing-registry[Exposing a default registry manually]
 * link:https://plugins.jetbrains.com/plugin/16476-knative\--serverless-functions-by-red-hat[Marketplace page for the Intellij Knative plugin]
 * link:https://marketplace.visualstudio.com/items?itemName=redhat.vscode-knative&utm_source=VSCode.pro&utm_campaign=AhmadAwais[Marketplace page for the Visual Studio Code Knative plugin]
-* link:https://docs.openshift.com/container-platform/latest/applications/creating_applications/odc-creating-applications-using-developer-perspective.html#odc-creating-applications-using-the-developer-perspective[Creating applications using the Developer perspective]
+* link:https://docs.openshift.com/container-platform/latest/applications/creating_applications/odc-creating-applications-using-developer-perspective.html#odc-creating-applications-using-the-developer-perspective[Creating applications]
```

Comment:
Yeah, probbly best to not tag "by using the Developer perspective" as this part of the heading should be deleted now that the admin and dev perspectives are merged.